packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt

```diff
@@ -183,7 +183,7 @@ internal class DataGripMongoDbDriver(
                 } else {
                     val decoderContext = DecoderContext.builder().build()
                     val outputCodec = codecRegistry.get(resultClass.java)
-                    val gson = Gson()
+                    val gson = GsonBuilder().serializeNulls().create()
```

Comment:
This is also a drive by, just to make sure we get fields with nulls as well in our parsed schema.

Reply:
Do we need null fields here? I'm fine keeping this for safety, but we should be already supporting partial documents, because this will deserialize fields that come `null` from MongoDB, but fields that are not in the document (most common case) won't be deserialized.

Reply:
yea this was only for the fields that were explicitly null in the document. Earlier, those fields were not resulting in the end schema.

Reply:
I think it's fine, but if we have a sample of 50 documents where the value is just `null`, how will the type look like? It will be BsonAny | BsonNull?